shell/common/shorebird.cc

```diff
@@ -69,6 +100,18 @@ void ConfigureShorebird(std::string cache_path,
     app_parameters.original_libapp_paths = c_paths.data();
     app_parameters.original_libapp_paths_size = c_paths.size();
 
+    // If we don't believe that we're running a full Flutter app, warn the user
+    // and don't try to initialize Shorebird.
+    std::string app_library_filename =
```

Comment:
Could this code be moved up so that it's executed earlier?

Reply:
Good call, fixed